src/YesSql.Core/Services/DefaultQuery.cs

```diff
@@ -1170,6 +1170,9 @@ public async Task<int> CountAsync(CancellationToken cancellationToken = default)
             }
         }
 
+        public Task<int> CountAsync()
+            => CountAsync(CancellationToken.None);
```

Comment:
All these method should be removed from every implementation and directly added into the interface.